src/app/home/styles/hero.js

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components';
+
+export const PrimaryBox = styled.div`
+    position: relative;
+    text-align: center;
+    color: white;
+`;
+
+export const HeroImage = styled.img`
+    object-fit: cover;
+    min-height: 90vh;
+    height: 92vh;
+    width: 100%;
```

Comment:
Hero ends before end of window. You can solve it by adding height: calc(100vh - 55px) to PrimaryBox, removing min-height from HeroImage and changing height to 100%.
![captura de pantalla_2018-10-27_22-08-05](https://user-images.githubusercontent.com/18080513/47610833-635d9780-da35-11e8-93b7-758889bc8712.png)